lib/feature_map/private/test_pyramid_file.rb

```diff
@@ -13,19 +13,9 @@ class FileContentError < StandardError; end
 
       FEATURES_KEY = 'features'
 
-      def self.write!(unit_examples, integration_examples, regression_examples, regression_assignments)
+      def self.write!(unit_by_feature, integration_by_feature, regression_by_feature)
```

Comment:
This class got slimmed down nicely and is now much more cleanly focused on the test pyramid output file. Great stuff! 💪